### PR TITLE
Fix several splunk errors

### PR DESCRIPTION
**Pull request by polishq**
#### What problem is this solving?

This PR fixes several problems spotted in Splunk:

- in the `getStores` query, when checking if the sitemap has already been generated, the code was sometimes unable to destructure `alreadyHasSitemap` from `getResponse` because the latter variable was `null`. This PR gives it a default value of an empty object to allow the destructuring to proceed
- some stores were throwing an error if they had more than 10,000 pickup points, as the Logistics API will start throwing errors if records beyond the first 10,000 are requested (i.e. `?page=100` and beyond). This PR limits the app from requesting more than 10,000 stores
- errors were not being logged at the `error` level (no level was specified)
- accounts with large numbers of stores were causing the frontend layout to become absurdly tall; this PR adds a `max-height` to the stores list container div
- I also adjusted the frontend list component to show the Spinner sooner and changed the logic for the `loadAll()` function (previously it was called whenever a `strike` was generated, which seemed weird. Now it is only automatically called if the component gets to 4 `strike`s) 

Additionally, this PR adjusts the `service.json` values to ensure the maximum TTL of 60 minutes is being set, and adds an `info` log to allow usage per account to be assessed.

#### How to test it?

Linked here: https://arthur--cutis.myvtex.com/stores
and here: https://arthur--drimjuguetes.myvtex.com/stores

### CODE REVIEW

File: react/List.tsx [hunk 1]
```diff
@@ -98,97 +98,100 @@ const StoreList = ({
...
-                iconWidth={iconWidth}
-                iconHeight={iconHeight}
-              />
+  const stores =
+    data?.getStores?.items.sort((a, b) => {
+      if (a[sortBy] < b[sortBy]) {
+        return -1
+      }
+
+      if (a[sortBy] > b[sortBy]) {
```
Comment (whc105) [inline react/List.tsx:137]: We can make this a bit easier to read with if.. else if... else

File: node/graphql/index.ts [hunk 2]
```diff
@@ -126,15 +126,18 @@ export const resolvers = {
...
       if (paging?.pages > 1) {
         let i = 2
         const results = [] as any
 
-        while (i <= paging.pages) {
+        // API will return errors starting at ?page=100
+        const limitPagesTo99 = paging.pages > 99 ? 99 : paging.pages
+
+        while (i <= limitPagesTo99) {
           results.push(hub.getStores({ ...param, page: i }))
```
Comment (whc105) [inline node/graphql/index.ts:141]: This If, while can be done more elegantly with a for loop
Something like
For (let i = 2; i <= pageLim; i++)

**Review by whc105 — CHANGES_REQUESTED** [review]

File: react/List.tsx (hunk 1 above)
Comment (polishq) [reply react/List.tsx:143]: ESLint doesn't like that 🤷

File: node/graphql/index.ts (hunk 2 above)
Comment (polishq) [reply node/graphql/index.ts]: Done!

**Review by whc105 — DISMISSED** [review]
lgtm

**Review by whc105 — APPROVED** [review]

**Pull request merged**